acceptance/setup/aio/010_Install.rb

```diff
@@ -29,19 +29,30 @@
 
 step "Install puppetserver..." do
   if ENV['SERVER_VERSION']
-    install_puppetlabs_dev_repo(master, 'puppetserver', ENV['SERVER_VERSION'])
-    install_puppetlabs_dev_repo(master, 'puppet-agent', ENV['SHA'])
-    master.install_package('puppetserver')
+    server_version = ENV['SERVER_VERSION']
+    if master[:hypervisor] == 'ec2' && master[:platform].match(/(?:el|centos|oracle|redhat|scientific)/)
```

Comment:
Why not just make this the code to use if we want nightly? i.e. SHA=nightly SERVER_VERSION=5 or something. Having it specific to the ec2 instance seems a bit odd.